src/App.jsx

```diff
@@ -14,45 +16,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const sortMethod = {
+  Alphabet: 'alphabetical',
+  Length: 'length',
+};
+
+function getSortedList(goods, { sortOrder, isReversed }) {
+  const preparedGoods = [...goods];
+
+  switch (sortOrder) {
+    case sortMethod.Length:
+      preparedGoods.sort((a, b) => a.length - b.length);
+      break;
+
+    case sortMethod.Alphabet:
+      preparedGoods.sort((a, b) => a.localeCompare(b));
+      break;
+
+    default:
+      break;
+  }
+
+  if (isReversed) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
+export const App = () => {
+  const [sortOrder, setSortOrder] = useState('');
+  const [isReversed, setIsReversed] = useState('');
```

Comment:
`isReversed` has type boolean, so it is not very good that by default it is a string